tools/gce-quota-sync/README.md

```diff
@@ -0,0 +1,116 @@
+# GCE quota exporter
+
+This tool serves a single, simple purpose: computing [GCE resource
+quota](https://cloud.google.com/compute/quotas) usage and exporting it to
+Stackdriver as [custom
+metrics](https://cloud.google.com/monitoring/custom-metrics/), so that alert
+rules can then be set to proactively monitor specific (or all) quotas.
+
+It does this by:
+
+* interfacing with the GCE API, and calling the
+  [`projects.get`](https://cloud.google.com/compute/docs/reference/rest/v1/projects/get)
+  and
+  [`regions.get`](https://cloud.google.com/compute/docs/reference/rest/v1/regions/get)
+  methods for global and regional quotas
+* deriving current utilization (usage / limit) for each quota, then converting
+  it into a time series
+* writing the time series to Stackdriver in the
+  `custom.googleapis.com/quota/gce` metric
+
+
+## Usage
```

Comment:
You should mention which versions of Python are supported, e.g. 2.7 and 3.4+, only 2.7, only 3.5+, etc.